ports/libedit/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "libedit",
+  "version-date": "2024-08-08",
+  "description": "A command line editor library that provides generic line editing, history, and tokenization functions, similar to those found in GNU Readline.",
+  "homepage": "https://thrysoee.dk/editline/",
+  "license": "BSD-4-Clause-UC",
+  "supports": "linux"
```

Doesn't it support osx or mingw or BSDs?

---

They don't appear to publicise supported operating systems and I don't have an OSX or BSD system to test it on, so it's difficult for me to confirm one way or another

---

Well, vcpkg CI has at least osx and android, and the point of autotoolizing the pristine BSD source is ... portability.

If we think that mingw should work and MSVC doesn't:
```suggestion
  "supports": "!windows | mingw"
```